dbschema.sql

```diff
@@ -8,55 +7,70 @@ CREATE TABLE LICCB.users(
     userEnabled BOOLEAN NOT NULL
 );
 
-CREATE TABLE LICCB.history(
-    participantID   CHAR(36) NOT NULL PRIMARY KEY,
-    firstName       VARCHAR(30) NOT NULL,
-    lastName        VARCHAR(30) NOT NULL,
-    eventCount      INT DEFAULT 0 NOT NULL,
-    cancellations   INT DEFAULT 0 NOT NULL,
-    lateCancel      INT DEFAULT 0 NOT NULL,
-    volunteerCount  INT DEFAULT 0 NOT NULL,
-    thumbsCount     INT DEFAULT 0 NOT NULL,
-    notes           TEXT
-);
-
 CREATE TABLE LICCB.events(
+    -- IDs
     eventID         CHAR(36) NOT NULL PRIMARY KEY,
+    managerID       CHAR(36) NOT NULL,              -- 'userID' of the user assigned to the event
+    creatorID       CHAR(36) NOT NULL,              -- 'userID' of the user who created the event
+
+    -- Event Details
     eventName       VARCHAR(100) NOT NULL,
-    manager         CHAR(36) NOT NULL,
-    creatorID       CHAR(36) NOT NULL,
-    capacity        INT NOT NULL,
     maxPartySize    INT NOT NULL,
     privateEvent    BOOLEAN NOT NULL,
     startTime       DATETIME NOT NULL,
     endTime         DATETIME NOT NULL,
-    staffRatio      FLOAT NOT NULL,
-    published       BOOLEAN NOT NULL,
-    eventNotes      TEXT,
+    eventStatus     ENUM('Unpublished', 'Registration Open', 'Registration Closed', 'Cancelled', 'Selection Finished'),
+    eventDesc       TEXT,
     eventMetadata   TEXT,
-    FOREIGN KEY (manager)
+
+    -- Foreign key for manager to ensure it is a real user
+    FOREIGN KEY (managerID)
         REFERENCES users(userID),
+
+    -- Foreign key for the creator to ensure it is a real user
     FOREIGN KEY (creatorID)
         REFERENCES users(userID)
 );
 
-CREATE TABLE LICCB.`event123e4567-e89b-12d3-a456-556642440000`(
-    participantID   CHAR(36) NOT NULL PRIMARY KEY,
-    partyID         CHAR(36) DEFAULT NULL,
-    isAdult         BOOLEAN NOT NULL,
+CREATE TABLE LICCB.participants(
+    -- IDs
+    participantID   CHAR(36) NOT NULL,
+    partyID         CHAR(36) DEFAULT NULL,  -- 'participantID' of the party leader
+    eventID         CHAR(36) NOT NULL,      -- Event the participant is registered for
+
+    -- Contact Info and Details
+    firstName       VARCHAR(30) NOT NULL,
+    lastName        VARCHAR(30) NOT NULL,
     phone           VARCHAR(30) NOT NULL,
     email           VARCHAR(75) NOT NULL,
     emergencyPhone  VARCHAR(30) NOT NULL,
     emergencyName   VARCHAR(60) NOT NULL,
+
+    -- User Submitted Selection Information
+    isAdult         BOOLEAN NOT NULL,
     hasCPRCert      BOOLEAN NOT NULL,
-    regTime         DATETIME,
```

Comment:
Why remove registration time? This is important for the first come first serve selection.